src/components/MetadataEditor/MetadataEditor.js

```diff
@@ -176,9 +176,9 @@ class MetadataEditor extends Component {
               flex: 1,
             }}
           >
-            <Button disabled={!onDelete} onClick={onDelete}>
+            {/* <Button disabled={!onDelete} onClick={onDelete}>
               <Delete /> Delete
-            </Button>
+            </Button> */}
```

Comment:
Still in source control, can just delete it

Reply:
I'm not sure the client asked for this removal. I keep it here as a reminder to ask.